carbon-design-system/carbon
# Time picker - New states and icons

Closes #13647

**Description**

- Added a new state called warning.
- Added icons to the current `invalid` state and to the new `warning` state.

**New**

Added a new state called warning.

**Testing**

- Go to the Time Picker Playground and test the `invalid` and `warning` state to check if they are working as expected.
- Check if the messages are being displayed correctly
